lib/osv/ecosystems.py

```diff
@@ -318,6 +322,7 @@ class Debian(Ecosystem):
   _API_PACKAGE_URL = 'https://snapshot.debian.org/mr/package/{package}/'
   _BACKOFF_FACTOR = 5
   _RETRY_TOTAL = 7
+  _REDIS_TTL_SECONDS = 3600
```

Comment:
We can make this even longer, like 6 hours.

Also let's break this up into

```python
6 * 60 * 60  # 6 hours.
```

To make the duration clearer to readers.